doc/start/intro.rst

```diff
@@ -28,8 +28,9 @@ required when running Ceph Filesystem clients.
   state of the Ceph cluster, including storage utilization, current
   performance metrics, and system load.  The Ceph Manager daemons also
   host python-based plugins to manage and expose Ceph cluster
-  information, including a web-based `dashboard`_ and `REST API`_.  At
-  least two managers are normally required for high availability.
+  information, including a web-based :doc:`/mgr/dashboard` and
```

Comment:
in the future, please do not link docs using paths, use the `:ref:` because it is dynamic. If the /mgr/dashboar document ever moves, this will mean the link will break

Reply:
Ok I will take care. Thanks.

Reply:
Thanks for pointing this out @alfredodeza - good point. @joscollin can you submit a separate PR to improve this? Thanks!

Reply:
Yes, I was planning to do that today. Just reached home now. I will submit a pr in a few hours.